### PR TITLE
Gherkin Scripts Covering Usability Enhancements for Eventing - [ODC-3953]

### PR DESCRIPTION
**Pull request by gajanan-more**
Fixes: https://issues.redhat.com/browse/ODC-4349

Problem: Gherkin Scripts Design for Usability enhancements for Eventing epic ODC-3953

Solution: These are the gherkin scripts designed covering all the scenarios related to Usability enhancements for Eventing epic ODC-3953

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/Knative/usability-Enhancement-Eventing-ODC-3953.feature [hunk 1]
```diff
@@ -0,0 +1,71 @@
+Feature: Usability Enhancement for Eventing
+    User should be able to select BootStrapServers and Topics from provided options
+
+
+Background:
+   Given cluster with OpenShift Serverless Operator installed
+   And Kafka Operator installed
```
Comment (invincibleJai) [inline frontend/packages/dev-console/integration-tests/features/Knative/usability-Enhancement-Eventing-ODC-3953.feature:7]: we may need to create CR for `KEK` as well

File: frontend/packages/dev-console/integration-tests/features/Knative/usability-Enhancement-Eventing-ODC-3953.feature [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    And user clicks on Event Sources card
+    And user clicks on Kafka Sources
+    And user enters the BootStrapServer name
+    And user enters the Topic name
+    And user selects Sink
+    And user selects Application
+    And user enters name for the Kafka Source
+    And user clicks on Create button
+    Then user gets redirected to Topology page 
+    And user can see the Kafka source created
```
Comment (invincibleJai) [inline frontend/packages/dev-console/integration-tests/features/Knative/usability-Enhancement-Eventing-ODC-3953.feature:63]: do we think we need different scenario text here `Select multiple BootStrapServers and Topics` as one is just selection and another one is typing

File: frontend/packages/dev-console/integration-tests/features/Knative/usability-Enhancement-Eventing-ODC-3953.feature (hunk 1 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/Knative/usability-Enhancement-Eventing-ODC-3953.feature:7]: User has installed the Serverless Operator and kafka operator that means user has done all the prerequisites like creating serving, eventing and CR for Kafka.

File: frontend/packages/dev-console/integration-tests/features/Knative/usability-Enhancement-Eventing-ODC-3953.feature (hunk 2 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/Knative/usability-Enhancement-Eventing-ODC-3953.feature:63]: One scenario is for selecting whatever we have in the drop down menu and the other is entering the bootstrapserver and topics. We can send the String there, that's why I have added the separate scenario there.

**Comment by makambalaji** [comment]
Already reviewed by jai and got approved as well.. So I am lgtm for merge
/lgtm

**Pull request merged**